components/basic-transformers/src/main/java/org/datacleaner/beans/numbers/GenerateIdTransformer.java

```diff
@@ -38,35 +38,48 @@
 @Categorized(NumbersCategory.class)
 public class GenerateIdTransformer implements Transformer {
 
-	@Configured
-	@Description("A column which represent the scope for which the ID will be generated. "
-			+ "If eg. a source column is selected, an ID will be generated for each source record. "
-			+ "If a transformed column is selected, an ID will be generated for each record generated that has this column.")
-	InputColumn<?> columnInScope;
+    public enum IdType {
+        SEQUENCE, ROWID;
```

Comment:
And if you make your enum also implement HasName then they can have pretty names in the UI and documentation :-)